src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/MeshService.java

```diff
@@ -22,7 +22,7 @@
 @Slf4j
 public class MeshService {
 
-    private static final Set<WorkflowId> SUPPORTED_WORKFLOWS = Set.of(WorkflowId.PATHOLOGY, WorkflowId.SCREENING);
+    private static final Set<WorkflowId> SUPPORTED_WORKFLOWS = Set.of(WorkflowId.PATHOLOGY_3, WorkflowId.SCREENING);
```

Comment:
Do we need to add PATHOLOGY_2 here?
```suggestion
    private static final Set<WorkflowId> SUPPORTED_WORKFLOWS = Set.of(WorkflowId.PATHOLOGY_2, WorkflowId.PATHOLOGY_3, WorkflowId.SCREENING);
```

Reply:
Yes. Not sure why _that_ didn't cause integration tests to fail...